internal/testrunner/runners/system/runner.go

```diff
@@ -257,7 +260,7 @@ func (r *runner) Run(ctx context.Context, options testrunner.TestOptions) ([]tes
 	}
 	result = r.newResult(fmt.Sprintf("%s - %s", resultName, testConfig.Name()))
 
-	scenario, err := r.prepareScenario(ctx, testConfig, svcInfo, serviceOptions)
+	scenario, err := r.prepareScenario(ctx, testConfig, svcInfo)
```

Comment:
Take the change to move `createServiceOptions` within `prepareScenario` method.

From this object, it was just used `DataStreamRootPath` but this value is also available from the `runner` struct once `r.initRun()` has been called.